packages/clay-grid/src/ClayGrid.js

```diff
@@ -0,0 +1,137 @@
+import 'clay-card';
+import Component from 'metal-component';
+import defineWebComponent from 'metal-web-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import templates from './ClayGrid.soy.js';
+
+/**
+ * Metal ClayGrid component.
+ */
+class ClayGrid extends Component {
```

Comment:
hey @carloslancha, I think this name is much more generic than leaving it just for the listing of cards, we can give another impression by the name of the component. Maybe rename it to `ClayCardGrid`,` ClayCardList` or `ClayCardGroup`. What do you think about this?

Reply:
+1 `ClayCardGrid` my namespace on Clay was based on this item https://i5.walmartimages.com/asr/1fc1aeeb-c02f-4ea7-9ab7-7a0e757bd171_1.a8424a26d75bcc472dda2395f8e5fc77.jpeg